File: src/constants/financeConstants.js
export const FINANCE_MODULES = {
  supplier: "supplier",
  customer: "customer",
  expenses: "expenses",
  analysis: "analysis",
};

export const FINANCE_TABS = {
  customer: "musteri",
  supplier: "tedarikci",
};

export const PAYMENT_STATUSES = {
  pending: "pending",
  paid: "paid",
};

export const PAYMENT_STATUS_OPTIONS = [
  { value: PAYMENT_STATUSES.pending, label: "Bekleyen" },
  { value: PAYMENT_STATUSES.paid, label: "Ödenen" },
];

export const ALL_FILTER_VALUE = "all";

export const STATUS_KEYS = {
  all: ALL_FILTER_VALUE,
  paid: "paid",
  pending: "pending",
  approaching: "approaching",
  overdue: "overdue",
};

export const STATUS_FILTERS = {
  [STATUS_KEYS.all]: "Tümü",
  [STATUS_KEYS.paid]: "Ödenen",
  [STATUS_KEYS.pending]: "Bekleyen",
  [STATUS_KEYS.approaching]: "Yaklaşan",
  [STATUS_KEYS.overdue]: "Geciken",
};

export const CURRENCIES = {
  try: "TRY",
  eur: "EUR",
  usd: "USD",
};

export const CURRENCY_OPTIONS = [
  CURRENCIES.try,
  CURRENCIES.eur,
  CURRENCIES.usd,
];

export const DEFAULT_EXCHANGE_RATES = {
  [CURRENCIES.eur]: 1,
  [CURRENCIES.usd]: 1,
  [CURRENCIES.try]: 1,
};

File: backend-dna/src/DnaKalip.Api/Domain/FinanceConstants.cs
namespace DnaKalip.Api.Domain;

public static class FinanceTabs
{
    public const string Customer = "musteri";
    public const string Supplier = "tedarikci";
}

public static class PaymentStatuses
{
    public const string Pending = "pending";
    public const string Paid = "paid";
}

public static class FinanceStatusKeys
{
    public const string Pending = "pending";
    public const string Paid = "paid";
    public const string Approaching = "approaching";
    public const string Overdue = "overdue";
}

public static class FinanceStatusLabels
{
    public const string Pending = "Bekleyen";
    public const string Paid = "Ödenen";
    public const string Approaching = "Yaklaşan";
    public const string Overdue = "Geciken";
}

public static class Currencies
{
    public const string Try = "TRY";
    public const string Eur = "EUR";
    public const string Usd = "USD";
    public const string LegacyTry = "TL";
}

public static class ExchangeRateTypes
{
    public const string Fixed = "fixed";
    public const string Current = "current";
    public const string FixedTurkish = "sabit";
    public const string CurrentTurkish = "guncel";
    public const string CurrentTurkishWithAccent = "güncel";
}
